src/cs/LionWeb.Integration.WebSocket.Tests/SetupTrace.cs

```diff
@@ -0,0 +1,38 @@
+﻿// // Copyright 2024 TRUMPF Laser GmbH
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License");
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// //     http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+// //
+// // SPDX-FileCopyrightText: 2024 TRUMPF Laser GmbH
+// // SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+
+namespace LionWeb.Integration.WebSocket.Tests;
+
+[SetUpFixture]
+[Timeout(WebSocketTestBase.TestTimeout)]
+[NonParallelizable]
+public class SetupTrace
```

Comment:
For this kind of boilerplate code, it'd be really useful to document its purpose – probably a framework-related configuration – in ~1 sentence.

Also: it should be either `SetUpTrace` or `TraceSetup` 😺

Reply:
Most of this code, including the name, is "inspired" 1:1 from NUnit docs

Reply:
Can you add a link to those docs, or provide a 1 sentence summary?